test.php

```diff
@@ -0,0 +1,35 @@
+<?php
+require_once 'Task.php';
+
+$task = new Task(999, 555);
+
+var_dump($task->getAllActions());
+echo "\n";
+var_dump($task->getAllStatuses());
+echo "\n";
+echo "ClientID: " . $task->getClientId() . "\n";
+echo "PerformerID: " . $task->getPerformerId() . "\n";
+
+echo "Ожидается статус 'in_progress', получено: " . $task->getNextStatus($task::ACTION_ASSIGN) . "\n";
```

Comment:
для передачи констант используем обращение не от переменной `$task` а от имени класса `Task`, так логичнее и общепринято делать именно так:

```suggestion
echo "Ожидается статус 'in_progress', получено: " . $task->getNextStatus(Task::ACTION_ASSIGN) . "\n";
```